services/recommendations/main.py

```
@@ -1,77 +1,259 @@
+            logging.error(f"Error loading model: {e}")
+            raise RuntimeError("Failed to load recommendation model")
+    
+    # TODO: Improve prompt
+    def apply_prompt(self, d):
+
+        template = "Head line: {NewsTitle}\nSummery: {NewsTLDR}\nLiked: {Liked}"
+        apply = lambda data: template.format(
+            NewsTitle=data.NewsTitle,
+            NewsTLDR=data.NewsTLDR,
+            Liked=data.Liked
+        )
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `apply` as a `def`

(E731)